server/pkg/builtin/manifest.yml

```diff
@@ -1,3 +1,4 @@
+$schema: manifest.schema.json
```

Comment:
refering to `schemas/plugin_manifest.json` is not good? https://github.com/reearth/reearth/blob/main/server/schemas/plugin_manifest.json